app/src/main/res/layout/administrator_controls_profile_view.xml

```diff
@@ -33,14 +33,14 @@
       android:id="@+id/edit_profiles_text_view"
       style="@style/Body"
       android:layout_width="match_parent"
-      android:background="@color/admin_controls_options_background_color"
+      android:background="@color/white"
       android:onClick="@{(v) -> viewModel.onEditProfilesClicked()}"
-      android:paddingStart="16dp"
+      android:paddingStart="@dimen/administrator_controls_profile_view_down_below_text_view_padding_start"
       android:paddingTop="20dp"
-      android:paddingEnd="16dp"
+      android:paddingEnd="@dimen/administrator_controls_profile_view_down_below_text_view_padding_end"
```

Comment:
avoid using "lower upper below" adjectives rather use something like sub_heading_text_view/ option_text_view for example. i.e. you can use property/feature of that text view to define them instead of its relative position.